frontend/src/App.css

```
@@ -40,3 +40,324 @@
+    calc(var(--dash-pad-y) + env(safe-area-inset-bottom))
+    calc(var(--dash-pad-x) + env(safe-area-inset-left));
+  min-block-size: 100dvh;
+  background-color: var(--color-bg-theme, #f8fafc);
+}
+
+.dashboard-inner {
+  max-inline-size: 80rem;
+  margin-inline: auto;
+}
+
+/* ── Header ─────────────────────────────────────────────────────────────────*/
```

Stylelint (17.3.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)